runner/src/internal.rs

```diff
@@ -99,10 +101,33 @@ pub struct BuildClient {
     pub client_rust_target: Option<String>,
 }
 
+#[derive(Clone, Debug)]
```

Comment:
Not a big deal, but it seems this should impl `Display` instead of (or in addition to `Debug`) for consistency, but up to you.

Reply:
There's no `derive(Display)` and I didn't think it was worth a manual `impl`.